commit 8d7af820310cb547c4cb7bdc321c161833eba57f
Author: Kees Cook <kees@kernel.org>
Date:   Wed Oct 18 17:46:20 2023 -0700

    MAINTAINERS: Include stackleak paths in hardening entry
    
    While most of the gcc-plugins are self-contained in the
    scripts/gcc-plugins directory, stackleak actually has some additional
    files. Add those so changes are directed to the hardening list.
    
    Suggested-by: Mark Rutland <mark.rutland@arm.com>
    Acked-by: Mark Rutland <mark.rutland@arm.com>
    Link: https://lore.kernel.org/r/20231019004616.work.960-kees@kernel.org
    Signed-off-by: Kees Cook <keescook@chromium.org>

diff --git a/MAINTAINERS b/MAINTAINERS
index 5f18ed0fbd42..a4de29c0036f 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -8615,6 +8615,8 @@ L:	linux-hardening@vger.kernel.org
 S:	Maintained
 T:	git git://git.kernel.org/pub/scm/linux/kernel/git/kees/linux.git for-next/hardening
 F:	Documentation/kbuild/gcc-plugins.rst
+F:	include/linux/stackleak.h
+F:	kernel/stackleak.c
 F:	scripts/Makefile.gcc-plugins
 F:	scripts/gcc-plugins/